src/views/tables/PositionsTable.tsx

```diff
@@ -195,6 +195,14 @@ const getPositionsTableColumnDef = ({
                 value={unrealizedPnl.current}
                 showSign={ShowSign.Negative}
               />
+              <$OutputSigned
+                sign={getNumberSign(unrealizedPnl.current)}
+                type={OutputType.Percent}
+                value={unrealizedPnlPercent.current}
+                showSign={ShowSign.Negative}
+                fractionDigits={0}
```

Comment:
why no fractionDigits?

Reply:
I didn't like seeing so many decimal points and it doesn't feel as necessary here.

Reply:
it's quite noisy with 2 decimals

Reply:
okay. we can always expand to 2 decimals in the future. My first assumption is that 2 decimals for PnL is completely normal.